arrow/arrow.py

```diff
@@ -969,8 +969,50 @@ def humanize(
                 )
             )
 
-    # query functions
+    def dehumanize(self, humanized_str="now", locale="en_us"):
```

Comment:
`humanized_str` should be a required argument since it does not make much sense to duhumanize nothing.

Also, let's change the name of this variable to `humanized`.